src/test/unit/algorithms/ConnectionsPerformanceTest.cpp

```diff
@@ -95,16 +90,18 @@ float runTemporalMemoryTest(UInt numColumns, UInt w,   int numSequences,
 
   for (auto sequence : sequences) {
     for (auto sdr : sequence) {
-      _feedTM(tm, sdr, false);
-      tm.reset();
+      tm.compute(sdr, false, unused_, unused_);
     }
+    tm.reset();
```

Comment:
This is probably why it used to pass but doesn't any more.  

Before, it would reset the TM after every cycle so it never grew any synapses...
Now, it will properly learn the sequences which is far more time consuming.

This kind of error is why this test should also check the anomaly score.

It would be great if this test verified after the first pass through the data set that the anomaly score is high.
Then verify after completing the training that the anomaly score is low.

Reply:
> Then verify after completing the training that the anomaly score is low. 

this is what I wanted from the Hotgym example. I'll see if it's simple enough to be added to this test as well.